### ISSUE TITLE
User reported issue

### ISSUE THREAD

**Issue by justinsb** [kubernetes/kops#767]
From slack:

kubelet in kops is having trouble starting .. this seems to be the cause:
```
Oct 31 03:30:34 ip-172-25-109-160 kubelet[762]: E1031 03:30:34.711573     762 cni.go:163] error updating cni config: No networks found in /etc/cni/net.d
```

then there is
```
Oct 31 03:30:34 ip-172-25-109-160 kubelet[762]: W1031 03:30:34.714104     762 manager.go:144] Unable to connect to Docker: Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```

and in docker im seeing
```
Oct 31 03:30:47 ip-172-25-109-160 docker[1106]: time="2016-10-31T03:30:47.769538070Z" level=fatal msg="Error starting daemon: Error initializing network controller: Error creating default \"bridge\" network: bridge device with non default name cbr0 must be created manually"
```

Yeah ok. Ive tried about 6 terminations and it keeps happening. Im guess there is more to it.  The cloud-final service seems to fail:
```
● cloud-final.service                                 loaded failed     failed             LSB: Cloud init modules final jobs
```

My create looked like the following:

```
kops create cluster --cloud aws --dns-zone "test.com" --master-size "t2.medium" --master-zones "us-west-2a,us-west-2b,us-west-2c" --network-cidr "172.20.0.0/16" --node-count 1 --node-size "m4.xlarge" --out mycluster.test.com --ssh-public-key ~/key.pem --zones "us-west-2a,us-west-2b,us-west-2c" mycluster.test.com
```

**Comment by bernielomax** [comment]
Docker seems to work initially then falls over:
```
root@ip-172-25-52-24:~# docker ps -a
CONTAINER ID        IMAGE                                        COMMAND                  CREATED              STATUS                      PORTS               NAMES
b0e202fdea19        gcr.io/google_containers/kube-proxy:v1.4.4   "/bin/sh -c 'kube-pro"   14 seconds ago       Up 13 seconds                                   k8s_kube-proxy.49161270_kube-proxy-ip-172-25-52-24.us-west-2.compute.internal_kube-system_167e6c33de2ce8b0e7d747a802bd6411_d5d3cde4
9bb378abde65        gcr.io/google_containers/pause-amd64:3.0     "/pause"                 About a minute ago   Up 46 seconds                                   k8s_POD.d8dbe16c_kube-proxy-ip-172-25-52-24.us-west-2.compute.internal_kube-system_167e6c33de2ce8b0e7d747a802bd6411_23983ba8
2b54226abe19        kope/protokube:1.3                           "/usr/bin/protokube -"   37 minutes ago       Exited (0) 33 minutes ago                       infallible_chandrasekhar
root@ip-172-25-52-24:~# docker ps -a
An error occurred trying to connect: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1: read unix @->/var/run/docker.sock: read: connection reset by peer
```
I can also confirm that deleting /var/lib/docker and restarting does not fix the issue. The only way I have found to get docker to start is by installing docker v1.12.3

**Comment by bernielomax** [comment]
Changing the node instance type in the cluster to t2.medium seem to resolve the issue.

**Comment by bernielomax** [comment]
After working with @justinsb on this issue it seems that it was likely caused by creating the cluster with an older version of kops. After using version 1.4.1 to perform a `kops udpate`  and a `kops rolling-update` the cluster started behaving as expected. However during the initial rolling update (without the master interval param) the update appeared to hang after shutting down the first node. See issue https://github.com/kubernetes/kops/issues/768

**Comment by callmeradical** [comment]
Hey I am experiencing a similar issues standing up a brand new cluster. Every single node has a problem with docker 1.11.2 starting. As a result it can't run the kubelet or isn't running it.

**Comment by callmeradical** [comment]
Turns out my issue was exactly as @bernielomax described.

**Comment by justinsb** [comment]
Going to close as I think this is resolved anyway, and I think it is certainly going to be resolved by kops 1.5.0!
